gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,82 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    attr_accessor :appliance_pg_data, :standby_host
```

Comment:
On that note, maybe `ask_for_database_credentials should be reimplemented for these classes because the values have slightly different meanings. Maybe we shouldn't be subclassing `DatabaseConfiguration` at all. I think that would make what these classes are doing much easier to understand.